test/search-query-template.json

```diff
@@ -0,0 +1,5 @@
+{
+    "operationName":"searchResultItems",
+    "variables":{"input":[{"keywords":[],"filters":[{"property":"kind","values":["Pod"]}],"limit":10000}]},
+    "query":"query searchResultItems($input: [SearchInput]) {\n  searchResult: search(input: $input) {\n    items\n    __typename\n  }\n}"
+}
```

Comment:
do you think in this first round we can create a foundation for adding 1+ query

Reply:
I don't fully understand this request, but maybe this helps...
After loading this template, we could programmatically replace the `variables` json to create any type of search request.